src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c

```diff
@@ -1890,3 +1894,6 @@ MODULE_VERSION(XRT_DRIVER_VERSION);
 MODULE_DESCRIPTION(XOCL_DRIVER_DESC);
 MODULE_AUTHOR("Lizhi Hou <lizhi.hou@xilinx.com>");
 MODULE_LICENSE("GPL v2");
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(5,16,0)  || defined(RHEL_9_0_GE)
+MODULE_IMPORT_NS(DMA_BUF);
```

Comment:
Same question here for importing DMA_BUF

Reply:
Yes @sureshreddyavula , We are seeing the DKMS configure issue while installing XRT on 6.2 kernel.